yadm

```diff
@@ -155,22 +155,30 @@ function clone() {
 
   #; add the specified remote, and configure the repo to track origin/master
   debug "Adding remote to new repo"
-  git remote add origin "$1"
+  if [$GIT_PROGRAM = "hub"] ; then
+    "$GIT_PROGRAM" remote add origin "$@"
+  else
+    "$GIT_PROGRAM" remote add origin "$1"
+  fi
```

Comment:
This part is where we check if we're running hub and if we do, send in all arguments that the user supplied to the clone command.
```
yadm clone -p zimme/dotfiles
// Internal call
hub remote add origin -p zimme/dotfiles
```